lib/deprecated/pure/parseopt.nim

```diff
@@ -7,6 +7,12 @@
 #    distribution, for details about the copyright.
 #
 
+## **Note**: This module is deprecated since version 0.9.3.
```

Comment:
Nope, parseopt is here to stay. We need a posix compatible option parser in the stdlib at one day though.